packages/terra-clinical-data-grid/src/subcomponents/Cell.jsx

```diff
@@ -0,0 +1,139 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import memoize from 'memoize-one';
+
+import { KEYCODES } from '../utils/keycodes';
+
+import styles from './Cell.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * String identifier of the section in which the Cell will be rendered.
+   */
+  sectionId: PropTypes.string.isRequired,
+  /**
+   * String identifer of the row in which the Cell will be rendered.
+   */
+  rowId: PropTypes.string.isRequired,
+  /**
+   * String identifier of the column in which the Cell will be rendered.
+   */
+  columnId: PropTypes.string.isRequired,
+  /**
+   * String-formatted width that the Cell should be rendered as. Any valid css width value is supported (i.e. 200px, 3rem).
+   */
+  width: PropTypes.string.isRequired,
```

Comment:
Does this need to be uniformed across columns and rows? I'm wondering if this is passed from one of these columns or can this be user defined? (sorry if this obvious, initial PR walkthrough 😬 )

Reply:
The widths are defined by the columns, but the Cells also need to know that width when the render so they can set it as an inline style. These widths are the only things keeping the columns in alignment; since we're not using a <table>, there is no inherent alignment happening.

A Row will set its width to be the total of all the Cells rendered within it.

Reply:
It might be helpful to mention this would be passed through by its consuming component. We have done with with `intl` and `app` in some places and I think its useful to define the API but also note where it come from. For example, [Application Layout](https://github.com/cerner/terra-framework/blob/01daffb58489853b7663bd406ba0dbd53df96e46/packages/terra-application-layout/src/ApplicationLayout.jsx#L20)